.github/workflows/build-and-deploy.yaml

```diff
@@ -12,6 +12,7 @@ jobs:
     with:
       projectName: pennclubs
       path: backend
+      pythonVersion: 3.11-bookworm
```

Comment:
What's this `bookworm` stuff?

Reply:
`bookworm` is the latest stable Debian release and it seems that Docker has [removed support for `buster`](https://github.com/docker-library/python/pull/822) (which is two versions behind `bookworm`) with later versions of Python. Also if we don't specify the image it defaults to `python3.8-buster` which doesn't make sense now that we're using 3.11